src/services/paramValidation.spec.ts

```diff
@@ -145,15 +145,14 @@ test.each([
 })
 
 test.each([
-  { path: '/duplicate/[foo]', query: 'params=[?foo]' },
+  { path: '/duplicate/[foo]', host: 'https://router.kitbag.dev', query: 'params=[?foo]' },
```

Comment:
This test correctly type errored because you cannot create an external route without a host.